### PR TITLE
Added support Cat TKL PCB

### PR DESCRIPTION
**Pull request by HorrorTroll**
## Description

As per title.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/horrortroll/cat_tkl/info.json [hunk 1]
```diff
@@ -0,0 +1,993 @@
...
+    "bootloader": "rp2040",
+    "features": {
+        "bootmagic": true,
+        "mousekey": true,
+        "extrakey": true,
+        "console": false,
+        "command": false,
+        "nkro": true
+    },
+    "community_layouts": ["tkl_f13_ansi", "tkl_f13_ansi_split_bs_rshift", "tkl_f13_ansi_tsangan", "tkl_f13_ansi_tsangan_split_bs_rshift", "tkl_f13_iso", "tkl_f13_iso_split_bs_rshift", "tkl_f13_iso_tsangan", "tkl_f13_iso_tsangan_split_bs_rshift"],
```
Comment (drashna) [inline keyboards/horrortroll/cat_tkl/info.json:25]: Nitpick: 
```suggestion
    "community_layouts": [
        "tkl_f13_ansi", 
        "tkl_f13_ansi_split_bs_rshift", 
        "tkl_f13_ansi_tsangan", 
        "tkl_f13_ansi_tsangan_split_bs_rshift", 
        "tkl_f13_iso", 
        "tkl_f13_iso_split_bs_rshift", 
        "tkl_f13_iso_tsangan", 
        "tkl_f13_iso_tsangan_split_bs_rshift"
    ],
```

**Review by drashna — APPROVED** [review]

File: keyboards/horrortroll/cat_tkl/info.json (hunk 1 above)
Comment (HorrorTroll) [reply keyboards/horrortroll/cat_tkl/info.json]: Thanks, I've just pushed new commit to resolved this.

File: keyboards/horrortroll/cat_tkl/readme.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# Cat x(
+
+![Cat](https://i.imgur.com/sPZxgXuh.png)
+
+A TKL keyboard, which controlled by an RP2040 chipset.
```
Comment (dunk2k) [inline keyboards/horrortroll/cat_tkl/readme.md:5]:
```suggestion
# Cat TKL

![CatTkl](https://i.imgur.com/sPZxgXuh.png)

TKL (F13 top row) sized keyboard controlled by an RP2040 controller, with layout options:
- ANSI or ISO form factor
- Split or 2u Backspace
- Split (1.75u-1u) or 2.75u Right Shift
- Standard or Tsangan bottom row
```

**Review by dunk2k — CHANGES_REQUESTED** [review]

File: keyboards/horrortroll/readme.md [hunk 3]
```diff
@@ -13,6 +13,7 @@
   | Lemon40            | ATMEGA32U4 | 40% Alice | 0x7516    | 0x6000     | Solder, RGB Underglow, OLED 128x32 |
   | Caticorn (Solder)  | RP2040     | TKL       | 0x7516    | 0x6001     | Solder                             |
   | Caticorn (Hotswap) | RP2040     | TKL       | 0x7516    | 0x6002     | Hotswap                            |
+  | Cat x(             | RP2040     | TKL       | 0x7516    | 0x6003     | Solder                             |
```
Comment (dunk2k) [inline keyboards/horrortroll/readme.md:16]:
```suggestion
  | Cat TKL             | RP2040     | TKL       | 0x7516    | 0x6003     | Solder                             |
```

**Review by dunk2k — CHANGES_REQUESTED** [review]

File: keyboards/horrortroll/readme.md (hunk 3 above)
Comment (HorrorTroll) [reply keyboards/horrortroll/readme.md]: Thanks, I've just push a new commit to resolved this.

**Pull request closed without merging**